src/hooks/useCreateEditCoupon.ts

```diff
@@ -57,10 +64,29 @@ type UseCreateEditCouponReturn = {
 }
 
 const formatCouponInput = (values: CreateCouponInput | UpdateCouponInput) => {
-  const { amountCents, ...others } = values
+  const {
+    amountCents,
+    amountCurrency,
+    expirationDate,
+    percentageRate,
+    frequencyDuration,
+    ...others
+  } = values
 
   return {
-    amountCents: Math.round(Number(amountCents) * 100),
+    amountCents:
```

Comment:
You're doing this both here and in AddCouponCustomerDialog right ?

It could be nice to move it into serializers to avoid errors in case of refacto or BE changes